changed gpt sending reply method

mrozonn opened the pull request:
gpt sends reply in embed instead of message
unable to test for now because of open ai quota limit
additionally added "send_embed" command with long string to test character limits in embeds (to be deleted when gpt functionality is ready)

mrozonn committed:
changed sending gpt reply from message to embed

W.I.P, unable to test cause of gpt quota limits

ghost commented:
Embeds have limit of 4096, so this implementation is moving the goal post, not solving the issue. More work needs to be done to ensure expected behaviour from issue #18

mrozonn committed:
deleted send_embed command, minor cleanup

came to consensus that there is no need for implementing message splitting method, as ChatGPT will not generate text longer that 4096 characters due to limits
